config/config.go

```diff
@@ -113,17 +113,19 @@ type SignerProxyConfig struct {
 }
 
 type VerifierConfig struct {
-	Upstream        URL                          `yaml:"upstream"`
+	Upstream URL `yaml:"upstream"`
 	// Changed to string to be more JWT spec compliant - it can be either string or URL
 	Audience        string                       `yaml:"audience"`
 	CookiesEnabled  bool                         `yaml:"auth_cookies_enabled"`
+	CookiePath      string                       `yaml:"cookie_path"`
```

Comment:
Please consider adding the example values of new parameter to README.md https://github.com/eclipse/che-jwtproxy#verifier-config